Seminars/Sem.02/Pract.02/Task02.cpp

```diff
@@ -0,0 +1,34 @@
+//Gabriela Mladenova
+//SI
+//Group 1
+//FN: 3MI0600225
+
+#include <iostream>
+#include <fstream>
+
+const char FILENAME[] = "result.txt";
+
+size_t lengthOfFile(const char NAME[])
+{
+    std::ifstream file(NAME);
+    if (!file.is_open())
+    {
+        std::cout << "ERROR! The file could not be opened! ";
+        return 0;
+    }
+
```

Comment:
Save the initial position of tellg, in order not to change the position of tellg when finding the length of the file